### PR TITLE
Clarifications and minor formatting improvements

### PR DESCRIPTION
**Pull request by snoopdave**
I added some clarifications and tried to make this more of a step-by-step guide to installing the provisioner, based on my recent experience with setting it up on an OpenShift cluster

### CODE REVIEW

**Comment by snoopdave** [comment]
I just signed the CLA.

File: nfs-client/README.md [hunk 1]
```diff
@@ -2,37 +2,85 @@
...
+$ oadm policy add-scc-to-user hostmount-anyuid system:serviceaccount:default:nfs-client-provisioner
+$ oadm policy add-cluster-role-to-user nfs-client-provisioner-runner system:serviceaccount:default:nfs-client-provisioner
+```
+
+**Step 4: Configure the NFS-Client provisioner**
 
 Note: To deploy to an ARM-based environment, use: `deploy/deployment-arm.yaml` instead, otherwise use `deploy/deployment.yaml`.
-Modify `deploy/deployment.yaml` and change the values to your own NFS server:
 
+Next you must edit the provisioner's deployment file to add connection information for your NFS server. Edit `deploy/deployment.yaml` and replace the two occurances of <YOUR NFS SERVER HOSTNAME> with your server's hostname.
```
Comment (wongma7) [inline nfs-client/README.md]: spelling: s/occurances/occurrences/

Comment (snoopdave) [reply nfs-client/README.md]: Fixed. Thanks!

**Comment by snoopdave** [comment]
@wongma7 anything else I can do to get this merged?  I'm happy to help.

**Comment by wongma7** [comment]
/lgtm
thanks

**Pull request merged**